### PR TITLE
Improvements to line models

### PR DESCRIPTION
**Pull request by dietmarw**
These are the changes that @christiankral accidentally committed to the backport PR #3818.
I've marked them as draft for now.

### CODE REVIEW

File: Modelica/Electrical/Analog/Lines/M_OLine.mo [hunk 1]
```diff
@@ -218,7 +217,7 @@ public
             extent={{-150,90},{150,50}},
             textString="%name",
             textColor={0,0,255})}), Documentation(info="<html>
-<p>The segment_last model is part of the multiple line model. It describes the special  line segment which is used to get the line symmetrical as outlined in the M_OLine description. Using the loop possibilities of Modelica it is formulated by connecting components the number of which depends on the number of lines.</p>
+<p>The segment_last model is part of the multiple line model. It describes the special  line segment which is used to get the line symmetrical as outlined in the <code>M_OLine</code> description. Using the loop possibilities of Modelica it is formulated by connecting components the number of which depends on the number of lines.</p>
```
Comment (beutlich) [inline Modelica/Electrical/Analog/Lines/M_OLine.mo:220]:
```suggestion
<p>The segment_last model is part of the multiple line model. It describes the special line segment which is used to design the line symmetrical as outlined in the <code>M_OLine</code> description. Using the loop possibilities of Modelica it is formulated by connecting components the number of which depends on the number of lines.</p>
```

File: Modelica/Electrical/Analog/Lines/M_OLine.mo [hunk 2]
```diff
@@ -337,9 +336,9 @@ its current can be accessed for plotting.</p>
 <p>In the picture of the segment can be seen, that a single segment is asymmetric. 
 Connecting such asymmetric segments in a series forces also an asymmetric multi line. 
 To get a symmetric model which is useful for coupling and which guaranties the same pin properties,
```
Comment (beutlich) [inline Modelica/Electrical/Analog/Lines/M_OLine.mo:338]: * Sometimes it is called `symmetric`, sometimes `symmetrical`. Could be made consistent,
* `To get ...` could be changed to `To design ...` as proposed in the other comment.

File: Modelica/Electrical/Analog/Lines/M_OLine.mo [hunk 3]
```diff
@@ -348,7 +347,7 @@ the auxiliary segment_last. For the example with 4 lines the schematic of
 </blockquote>
 
 <p>The number of the capacitors and conductors depends on the number of single lines that are used,
-because each line is connected to every other line by both a capacitor and a conductor. 
+because each line is coupled to every other line by both a capacitor and a conductor.
```
Comment (beutlich) [inline Modelica/Electrical/Analog/Lines/M_OLine.mo:350]: If it a Modelica connect statement, I'd leave it as `connected`, which also is used in other descriptions.

File: Modelica/Electrical/Analog/Lines/M_OLine.mo [hunk 4]
```diff
@@ -408,7 +407,7 @@ for the <code>M_OLine</code> model, a sensible conductance-matrix would be:</p>
 </blockquote>
 
 <p>The user has the possibility to enable a conditional heatport. If so, the <code>M_OLine</code> 
-can be connected to a thermal network. When the parameter <code>alpha</code> is set to a value greater than zero,
+can be connected to a thermal network. When the parameter <code>alpha</code> is set to a value different than zero,
```
Comment (beutlich) [inline Modelica/Electrical/Analog/Lines/M_OLine.mo:410]:
```suggestion
can be connected to a thermal network. If the parameter <code>alpha</code> is set to a value different than zero,
```

**Review by AHaumer — APPROVED** [review]
Fine with me

**Comment by christiankral** [comment]
@dietmarw It's still a draft. Is it ready for review?

**Comment by dietmarw** [comment]
@christiankral  These are your changes so you can decide if it is still draft or not ;-)

**Review by christiankral — APPROVED** [review]
Alright then

**Pull request merged**